lib/src/modules/statusIndicatorCircle/statusIndicatorCircle.js

```diff
@@ -3,14 +3,12 @@ import cx from 'classnames';
 import Icon from '../../../Icon';
 
 export function StatusIndicatorCircle({ color, icon, solid, thickBorder }) {
-  const className = cx('status-indicator-circle', {
-    'thick-border': thickBorder,
-    'thin-border': !thickBorder
-  });
+  const className = cx('status-indicator-circle');
+  const boxShadow = `inset 0px 0px 0px ${thickBorder ? '4' : '2'}px ${color}`;
 
   const addedStyles = {
     backgroundColor: solid ? color : 'transparent',
-    borderColor: color
+    boxShadow
```

Comment:
Initially, we had a transition on the border width, this animation looked really choppy. It turns out shadow animations are much cleaner!